Aspire Budgeting/PreviewProvider/MockProvider.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  MockProvider.swift
+//  Aspire Budgeting
+//
+
+import Foundation
+
+struct MockProvider {
+  static var tabBarItems: [TabBarItem] {
+    var tabBarItems = [TabBarItem]()
+    tabBarItems.append(TabBarItem(imageName: "rectangle.grid.1x2", title: "Dashboard"))
+    tabBarItems.append(TabBarItem(imageName: "creditcard", title: "Accounts"))
+    tabBarItems.append(TabBarItem(imageName: "arrow.up.arrow.down", title: "Transactions"))
+    tabBarItems.append(TabBarItem(imageName: "gear", title: "Settings"))
+    return tabBarItems
+  }
+  
+//  static var cardViewItems: [CardView.CardViewItem] {
+//    var cardViewItems = [CardView.CardViewItem]()
+//    cardViewItems.append(.init(title: "Credit Card Payments",
+//  lowerBound: "$0",
+//  upperBound: "$1000",
+//  progressFactor: 0.25))
+//    cardViewItems.append(.init(title: "Fixed Expenses", lowerBound: "$0", upperBound: "$2500", progressFactor: 0.5))
+//    cardViewItems.append(.init(title: "Savings", lowerBound: "$0", upperBound: "$10000", progressFactor: 0.7))
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/845606672c945b4a26becac88e482112dc55b51f/Aspire Budgeting/PreviewProvider/MockProvider.swift#L25"></span>Line should be 100 characters or less: currently 112 characters<br />
<code>line_length</code> <code>MockProvider.swift:25</code></td>
    </tr>
  </tbody>
</table>